### PR TITLE
Begin tearing down QuickJS-support for bridging interfaces

### PR DESCRIPTION
**Pull request by swankjesse**
This is still general-purpose in native code. I plan to drop that in
a follow-up PR.

### CODE REVIEW

File: zipline/src/jniTest/kotlin/app/cash/zipline/testUtil.kt [hunk 1]
```diff
@@ -36,3 +37,11 @@ fun Zipline.loadTestingJs() {
     .use(BufferedReader::readText)
   quickJs.evaluate(testingJs, "testing.js")
 }
+
+operator fun <T : Any> QuickJs.set(name: String, type: KClass<T>, instance: T) {
```
Comment (swankjesse) [inline zipline/src/jniTest/kotlin/app/cash/zipline/testUtil.kt:41]: I’ve got to triage the tests. Some can be deleted, and some should be replaced with Zipline-layer tests. Once I’m done that triage I’ll delete these two functions.

File: zipline/src/jniTest/kotlin/app/cash/zipline/testUtil.kt [hunk 2]
```diff
@@ -36,3 +39,17 @@ fun Zipline.loadTestingJs() {
     .use(BufferedReader::readText)
   quickJs.evaluate(testingJs, "testing.js")
 }
+
+fun Throwable.stackTraceToString(): String {
+  val result = StringWriter()
+  printStackTrace(PrintWriter(result))
+  return result.toString()
+}
```
Comment (JakeWharton) [inline zipline/src/jniTest/kotlin/app/cash/zipline/testUtil.kt]: Relation to https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/stack-trace-to-string.html?

File: zipline/src/jniTest/kotlin/app/cash/zipline/QuickJsCompileTest.kt [hunk 3]
```diff
@@ -92,6 +93,7 @@ class QuickJsCompileTest {
     val value: String?
   }
 
+  @Ignore
```
Comment (JakeWharton) [inline zipline/src/jniTest/kotlin/app/cash/zipline/QuickJsCompileTest.kt:96]: What is the plan for these two tests? Replace with Zipline-equivalent? Or delete?

**Review by JakeWharton — APPROVED** [review]

File: zipline/src/jniTest/kotlin/app/cash/zipline/testUtil.kt (hunk 2 above)
Comment (swankjesse) [reply zipline/src/jniTest/kotlin/app/cash/zipline/testUtil.kt]: I should just use that thing.

File: zipline/src/jniTest/kotlin/app/cash/zipline/QuickJsCompileTest.kt (hunk 3 above)
Comment (swankjesse) [reply zipline/src/jniTest/kotlin/app/cash/zipline/QuickJsCompileTest.kt:96]: Check if they're covering anything not already covered. Only if they're doing something original do they survive.

**Pull request merged**